src/Search/Workflow/AbstractWorkflow.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace eLife\Search\Workflow;
+
+abstract class AbstractWorkflow
+{
+    abstract public function getSdkClass();
+
+    abstract public function index($entity);
```

Comment:
We could make this Model typehint: https://github.com/elifesciences/api-sdk-php/blob/master/src/Model/Model.php